how-it-works.html

```diff
@@ -0,0 +1,254 @@
+                Ocopea is able to do this thanks to two key ingredients:
+            </p>
+            <br/>
+            <ul class="list">
+                <li>Understanding the cloud native app topology</li>
+                <li>Abstraction of the service consumption API (Ocopea Brokers)</li>
+            </ul>
+            <br/>
+
+            <div class="title">Application topology</div>
+            <p>
+                In order for Ocopea to take application copies, it first needs to understand the application topology:
+                what components it is built of and what is the interaction between them. Ocopea differentiates between
+                two types of components: <span style="font-weight: bold">stateless components</span> (app service) and
+                <span style="font-weight: bold">stateful components</span> (infrastructure services).
+                Mostly thanks to the ability to easily scale stateless microservices this has become a common
+                distinction for cloud native apps.
+                In addition, while it is popular to run the stateless microservices using container orchestrators/PaaS -
+                many organizations prefer delegating the standard infrastructure services to
+                either SaaS (e.g. AWS RDS) or DevOps (e.g. Oracle database on premises).
+
+                <br/>
+                <br/>
+
+                Lets walk through an example:
+            </p>
+
+            <br/>
+            <br/>
+
+            <div class="image-holder">
+                <img src="assets/hackathon-app-template.png"
+                     width="517px"
+                     alt="Ocopea high level components"
+                />
+            </div>
+
+            <br/>
+            <br/>
+
+            <p>
+                The application topology diagram above describes our basic sample hackathon application.
+                The application is built of one app service: The "hackathon service", written in java delivered as a
+                docker container, and has two infrastructure service dependencies:
+                an S3 service for storing documents and a Postgres database for storing hackathon ideas metadata.
+                The hackathon service is the application entry point -
+                meaning application users can interact with it via the web.
+                <br/>
+                <br/>
+                In order to make a copy of the entire hackathon application state and later restore it on a different
+                environment,
+                Ocopea needs the ability to make copies of each individual component configuration and data:
+                <br/>
+            </p>
+            <ul class="list">
+                <li>
+                    The hackathon docker image version that is used including the configuration (scale, environment
```

Comment:
If I deploy an app via Ocopea and then scale up one of the microservices before making an app copy, will the same scale be applied to a restored copy?

Reply:
No, but it is a bug :)

Reply:
I can omit "scale" if you prefer, but I prefer opening a bug...

Reply:
If we have code that should scale but got broken somehow, fine - a bug.
If we never had it working, then we should remove scale.

Reply:
@mrmccarthy I have made the changes and squashed all commits (thanks for the typos fix 👍 ) please hit "approve" so I can merge